google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataSettings.java

```diff
@@ -219,6 +219,15 @@ public boolean isRefreshingChannel() {
     return stubSettings.getPrimedTableIds();
   }
 
+  /**
+   * Gets if latency based throttling is enabled for {@link
+   * BigtableDataClient#newBulkMutationBatcher(String)}
+   */
+  @BetaApi("Latency based throttling is not currently stable and may change in the future")
+  public boolean isLatencyBasedThrottlingForBatchMutationsEnabled() {
+    return stubSettings.bulkMutateRowsSettings().isLatencyBasedThrottlingEnabled();
+  }
+
```

Comment:
I'm not sure if this getter is necessary, but if you think it is, please be consistent an expose the target latency as well

Reply:
Ok, I'm going to keep the getters here, because all the other settings seem to have getters too?